pkgs/kde/frameworks/extra-cmake-modules/default.nix

```diff
@@ -1,8 +1,16 @@
-{mkKdeDerivation}:
+{ mkKdeDerivation
+, python3
+}:
 mkKdeDerivation {
   pname = "extra-cmake-modules";
 
-  outputs = ["out"];
+  outputs = [ "out" ];
+
+  # Packages that have an android APK (e.g. KWeather) require Python3 at build time.
+  # See: https://invent.kde.org/frameworks/extra-cmake-modules/-/blob/v6.1.0/modules/ECMAddAndroidApk.cmake?ref_type=tags#L57
+  extraPropagatedBuildInputs = [
```

Comment:
Should this be extraPropagatedNativeBuildInputs? I am not sure

Reply:
We don't have that, but you can set `propagatedNativeBuildInputs` directly.

Reply:
nixpks-review just finished and successfully built everything with this commit.

Not sure if `propagatedNativeBuildInputs` could potentially cause issues with setup hooks?

Reply:
It _shouldn't_, but cross is broken hard anyway so it's hard to tell...

Reply:
I don't have an opinion on this, so I'll leave it to you @K900 : `propagatedNativeBuildInputs` yay or nay?

Reply:
I can run nixpkgs-review on it tomorrow.

Reply:
Alright, I switched it over for now. lmk if I should switch it back.